src/services/expenses.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+let expenses = [];
+
+function getAll() {
+  return expenses;
+}
+
+function findExpenseById(expenseId) {
```

Comment:
function should filter expenses by some conditions (for example, return all expenses between dates)

Reply:
But why? Neither task, nor documentation requires that. This function only checks if the expense with given id, exists or not.